pkg/aggregator/apiserver/plugin/handler.go

```diff
@@ -64,7 +64,7 @@ func (h *PluginHandler) registerRoutes() {
 		case aggregationv0alpha1.DataSourceProxyServiceType:
 			// TODO: implement in future PR
 		case aggregationv0alpha1.QueryServiceType:
-			h.mux.Handle(proxyPath("/namespaces/{namespace}/connections/{uid}/query"), h.QueryDataHandler())
+			h.mux.Handle(proxyPath("/namespaces/{namespace}/datasources/{uid}/query"), h.QueryDataHandler())
```

Comment:
@toddtreece FYI... note that the URL is chaning in this PR (but the connections flavor will stay for a little while longer)